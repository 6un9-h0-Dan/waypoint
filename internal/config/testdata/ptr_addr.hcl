project = "test"

app "web" {
    url {}
}
