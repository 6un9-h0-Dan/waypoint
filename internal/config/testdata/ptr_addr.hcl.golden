(config.Config) {
 Runner: (*config.Runner)(<nil>),
 Project: (string) (len=4) "test",
 Apps: ([]*config.App) (len=1 cap=1) {
  (*config.App)({
   Name: (string) (len=3) "web",
   Path: (string) "",
   Labels: (map[string]string) <nil>,
   URL: (*config.AppURL)({
    AutoHostname: (*bool)(<nil>)
   }),
   Build: (*config.Build)(<nil>),
   Platform: (*config.Operation)(<nil>),
   Release: (*config.Operation)(<nil>)
  })
 },
 Labels: (map[string]string) <nil>
}
